roles/variables/vars/main.yml

```diff
@@ -180,8 +180,8 @@ kafka_controller_properties:
     enabled: "{{ zookeeper_client_authentication_type in ['kerberos', 'digest'] and kraft_migration|bool }}"
     properties:
       zookeeper.set.acl: 'true'
-  migration_broker_listener:
-    enabled: "{{ sasl_protocol=='plain' and kraft_migration|bool }}"
+  broker_listener:
+    enabled: "{{ sasl_protocol=='plain' }}" # might need to add for kerberos also in later release
```

Comment:
Since we are not adding kerberos currently does it mean for 7.6.0 kraft mode kerberos auth won't work ? If so will the communication be working without authentication or such setup would fail?

Reply:
no, it works as expected. the only case that doesn't work is if Controller has say sasl plain and broker has kerberos. We do not support different sasl by default. For such combinations, customer will have to use custom_properties

Reply:
The reason to not add kerberos jaas config rn is it currently works as expected so we want to make minimum changes as to not have to test everything again and avoid any last minute failure. In the long run, we should also add kerberos jaas config here for a more generic solution

Reply:
got it, makes sense